python/llm/src/bigdl/llm/transformers/convert.py

```diff
@@ -211,6 +211,11 @@ def _replace_with_low_bit_linear(model, qtype, modules_to_not_convert=None,
             if (not any(key in ".".join(current_key_name) for key in modules_to_not_convert) and
                     not isinstance(module, LowBitLinear)):
                 in_features, out_features, mp_group = linear_args
+                optimize_lm_head = False
+                if name == "lm_head":
```

Comment:
Not all models have name lm_head? seems chatglm has name output_layer?

Reply:
Yes, we need to check both seq_len dim and the name of lm_head before including new models.